src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="header-nav">
+          <div class="logo">
+            <a href="#">
+              <img src="./images/logo.png" alt="Moyo">
+            </a>
+          </div>
+          <nav class="nav">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a href="#" class="nav-list__link is-active">APPLE</a>
+              </li>
+              <li class="nav-list__item">
+                <a href="#" class="nav-list__link ">SAMSUNG</a>
+              </li>
```

Comment:
```suggestion
              <li class="nav-list__item">
                <a href="#" class="nav-list__link is-active">APPLE</a>
              </li>
              
              <li class="nav-list__item">
                <a href="#" class="nav-list__link ">SAMSUNG</a>
              </li>
```
It's better to add a blank line between multiple siblings.